apps/prairielearn/elements/pl-string-input/pl-string-input.mustache

```diff
@@ -39,8 +39,9 @@
                 autocomplete="off"
                 autocorrect="off"
                 {{^editable}}disabled{{/editable}}
-                aria-describedby="pl-string-input-{{uuid}}-suffix"
                 placeholder="{{placeholder}}"
+                {{#suffix}}aria-describedby="pl-string-input-{{uuid}}-suffix"{{/suffix}}
+                {{^label}}aria-label="Variable {{name}}"{{/label}}
```

Comment:
Same comment as elsewhere. I'll stop the duplicate comments after this, but my notes apply to every fallback that was added.

Reply:
785357da395b470176b257d2ad9cac4482f141ef